bitnami/postgresql/values.yaml

```diff
@@ -310,16 +314,21 @@ extendedConfConfigMap: ""
 initdbScripts: {}
 ## @param initdbScriptsConfigMap ConfigMap with scripts to be run at first boot
 ## NOTE: This will override initdbScripts
+##
 initdbScriptsConfigMap: ""
 ## @param initdbScriptsSecret Secret with scripts to be run at first boot (in case it contains sensitive information)
 ## NOTE: This can work along initdbScripts or initdbScriptsConfigMap
+##
 initdbScriptsSecret: ""
 ## @param initdbUser Specify the PostgreSQL username to execute the initdb scripts
 ##
 initdbUser: ""
 ## @param initdbPassword Specify the PostgreSQL password to execute the initdb scripts
 ##
 initdbPassword: ""
+
+containerPorts:
+  postgresql: 5432
```

Comment:
Metadata for this parameter is pending. Can you regenerate the README once added it?

Reply:
Added